sys/compat/freebsd32/syscalls.conf

```diff
@@ -14,6 +14,7 @@ abi_long="int32_t"
 abi_u_long="uint32_t"
 abi_semid_t="int32_t"
 abi_size_t="uint32_t"
+abi_intptr_t="intptr32_t"
```

Comment:
I don't see what benefit the extra type gives over just saying int32_t?

Reply:
It avoids all the extra gunk due to isptrtype() matching on non-pointer instances.  I didn't see an easy way to fix it in makesyscalls.lua.